src/sidebar/components/search-status-bar.js

```diff
@@ -21,16 +21,14 @@ const countVisibleAnns = annThread => {
  * A bar where the user can clear a selection or search and see whether
  * any search results were found.
  * */
-function SearchStatusBar({
-  selectedTab,
-  totalAnnotations,
-  totalNotes,
-  rootThread,
-}) {
+function SearchStatusBar({ rootThread }) {
   const storeState = useStore(store => store.getState());
   const clearSelection = useStore(store => store.clearSelection);
   const filterQuery = storeState.filterQuery;
   const filterActive = !!storeState.filterQuery;
+  const countAnnotations = useStore(store => store.countAnnotations());
+  const countNotes = useStore(store => store.countNotes());
+  const selectedTab = storeState.selectedTab;
```

Comment:
It is more efficient to extract only the data you need from the store in a `useStore` call rather than fetching the entire state object. Extracting only the data you need enables `useStore` to limit re-rendering of a component to only when the relevant state has changed.

Reply:
I looked at this in the search-status-bar pr and i had trouble determining which state the rootThread needed but I just looked at it again and for whatever reason it's clearer now so I'll go ahead and limit the states that this needs now.